docs/src/fs.rst

```diff
@@ -258,6 +258,13 @@ API
 
     Equivalent to :man:`utime(2)` and :man:`futime(2)` respectively.
 
+.. c:function:: int uv_fs_utime_ex(uv_loop_t* loop, uv_fs_t* req, const char* path, double btime, double atime, double mtime, uv_fs_cb cb)
+.. c:function:: int uv_fs_futime_ex(uv_loop_t* loop, uv_fs_t* req, uv_file file, double btime, double atime, double mtime, uv_fs_cb cb)
+
+    Equivalent to :c:func:`uv_fs_utime` and :c:func:`uv_fs_futime` unless you
+    are on OS X or Windows in which case these variants allow you to also set
+    the birth/creation time.
+
 .. c:function:: int uv_fs_link(uv_loop_t* loop, uv_fs_t* req, const char* path, const char* new_path, uv_fs_cb cb)
```

Comment:
Needs a ".. versionadded". @bnoordhuis this adds a new double to `uv_fs_t`, not sure we can land it on v1.x.